### DISCUSSION TITLE
how can i debug coffee-editor's backend?

### DISCUSSION THREAD

**Discussion by alanala** [eclipse-emfcloud/emfcloud#224] [Coffee Editor]
Hello, 

I follow the readme step:
'''
Install Eclipse
Import projects from backend
Set target org.eclipse.emfcloud.coffee.target.target
All backend applications are bundles as eclipse products and can be found in org.eclipse.emfcloud.coffee.product
'''
but when "Validate Plug-ins", shows the error:
Missing Constraint: Import-Package:org.eclipse.jetty.util.annotation; version="[9.4.44,10.0.0)"
Missing Constraint: Import-Package:org.eclipse.jetty.util.component; version="[9.4.44,10.0.0)"
Missing Constraint: Import-Package:org.eclipse.jetty.util.log; version="[9.4.44,10.0.0)"
Missing Constraint: Import-Package:org.eclipse.jetty.util.resource; version="[9.4.44,10.0.0)"
but all these are classes in the org.eclipse.jetty.util jar file which one is imported in the project,  i cannot see how to solve it
can you help me?

**Comment by eneufeld** [comment]
Hi @alanala,

Sorry for the late reply.
I will check whether I can reproduce this.
In the meantime did you try using the docker image?

Best,
Eugen

**Comment by eneufeld** [comment]
I called the build:backend step from root on main and it works.
Can you explain where and what is not working for you?
